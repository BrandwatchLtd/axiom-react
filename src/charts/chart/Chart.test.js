import React from 'react';
import renderer from 'react-test-renderer';
import { Chart } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <Chart { ...props } />
  );
}

describe('Chart', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders with minimumHeight', () => {
    const component = getComponent({ minimumHeight: '10rem' });
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
