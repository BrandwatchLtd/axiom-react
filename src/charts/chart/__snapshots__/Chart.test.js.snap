// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart renders with defaultProps 1`] = `
<div
  className="ax-chart"
  style={
    Object {
      "minHeight": undefined,
    }
  }
/>
`;

exports[`Chart renders with minimumHeight 1`] = `
<div
  className="ax-chart"
  style={
    Object {
      "minHeight": "10rem",
    }
  }
/>
`;
